Setup/InstallSchema.php

```diff
@@ -0,0 +1,94 @@
+<?php
+namespace Yotpo\Yotpo\Setup;
+
+use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+
+/**
+ * Class InstallSchema
+ * @package Innovadeltech\Wishlist\Setup
+ */
+class InstallSchema implements InstallSchemaInterface
+{
+    /**
+     * Install script.
+     *
+     * @param SchemaSetupInterface $setup
+     * @param ModuleContextInterface $context
+     */
+     protected $_resource;
+
+    public function install(SchemaSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $installer = $setup;
+        $installer->startSetup();
+
+        $installer->endSetup();
+
+        $table = $installer->getConnection()->newTable(
+            $installer->getTable('yotpo_rich_snippets')
+        )->addColumn(
+                'rich_snippet_id',
+                \Magento\Framework\DB\Ddl\Table::TYPE_INTEGER,
+                null,
+                array('identity' => true, 'nullable' => false, 'primary' => true),
+                'Id'
+            )->addColumn(
+                'product_id',
+                \Magento\Framework\DB\Ddl\Table::TYPE_INTEGER,
+                null,
+                array('nullable' => false),
+                'Product Id'
+            )->addColumn(
```

Comment:
@billykov Do we need a store_id at all?
Shouldn't it be the same for all entries?